libbeat/docs/processors-using.asciidoc

```diff
@@ -1339,6 +1339,7 @@ the Kubernetes node.
 processors:
 - add_kubernetes_metadata:
     host: <hostname>
+    # It defaults to `KUBECONFIG` variable if present.
```

Comment:
can you update these comments to make sure it says `environment variable` instead? Just in case, to avoid confusion